feature/orchestrator/src/test/java/com/simprints/feature/orchestrator/cache/OrchestratorCacheIntegrationTest.kt

```diff
@@ -21,7 +21,6 @@ import org.junit.Ignore
 import org.junit.Test
 import org.junit.runner.RunWith
 
-@Ignore("Jackson behaves differently in testing than in production, to be fixed later")
```

Comment:
There is an issue when serializing the results data 
I enabled this test so that we fix the issue before merging the PR